lsp-completion.el

```diff
@@ -194,7 +198,7 @@ Return `nil' when fails to guess prefix."
     (lsp--position-to-point (lsp:range-start (lsp:text-edit-range text-edit?))))
    (t
     (-let* (((&CompletionItem :label :insert-text?) item)
-            (text (or insert-text? label))
+            (text (or (unless (lsp-falsy? insert-text?) insert-text?) label))
```

Comment:
this could be rewritten as `(if (lsp-falsy...`, here and on a few more places.

Reply:
Yeah, I'm purposely writing like this so we can add more term to the `or` clause` if the LSP protocol ever changes/adds new text. Do you think I can merge keep this?

Reply:
ok, it is not a big deal...